ftw/zipextract/events.py

```diff
@@ -0,0 +1,8 @@
+from interfaces import IFile
+from interfaces import IZip
+from Products.Five.utilities.marker import mark
```

Comment:
For letting an object provide an interface we should use `zope.interface.alsoProvides` instead of `mark`.

Reply:
Got rid of the interface altogether